jenkinsfile-runner-steward-image/scripts/elasticsearch-log-config-tests.sh

```diff
@@ -82,19 +97,29 @@ function _define_testdata() {
   declare -gr EXPECTED_FLUENTD_MANDATORY=$(cat <<"EOF"
 {
   "unclassified": {
-    "elasticSearchLogs": {
-      "elasticSearch": {
-        "elasticsearchWriteAccess": {
-          "fluentd": {
-            "bufferCapacity": 1098304,
-            "bufferRetentionTimeMillis": 1000,
+    "elasticsearchLogs": {
+      "elasticsearch": {
+        "saveAnnotations": false,
+        "writeAnnotationsToLogFile": false,
+        "splitMessagesLongerThan": null,
+        "eventWriterConfig": {
+          "fluentdEventWriter": {
```

Comment:
I don't see this key "fluentdEventWriter" in the Jenkins configuration yaml file once plugin is enabled.